frontend/src/features/admin/agencies/AgencyFilterBar.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import FilterBar from 'components/SearchBar/FilterBar';
+import { Col } from 'react-bootstrap';
+import { SelectOption, AutoCompleteText } from 'components/common/form';
+import { ILookupCode } from 'actions/lookupActions';
+import { IAgencyFilter } from 'interfaces';
+import useCodeLookups from 'hooks/useLookupCodes';
+
+interface IProps {
+  value: IAgencyFilter;
+  onChange: (value: IAgencyFilter) => void;
+  handleAdd: (value: any) => void;
+}
+
+export const AgencyFilterBar: React.FC<IProps> = ({ value, onChange, handleAdd }) => {
+  const mapLookupCode = (code: ILookupCode): SelectOption => ({
+    label: code.name,
+    value: code.id.toString(),
+  });
+
+  const lookupCodes = useCodeLookups();
+  const agencyLookupCodes = lookupCodes.getByType('Agency');
+  const agencyOptions = (agencyLookupCodes ?? []).map(c => mapLookupCode(c));
+
+  return (
+    <FilterBar<IAgencyFilter>
+      initialValues={value}
+      onChange={onChange}
+      searchClassName="bg-primary"
+      plusButton={true}
+      filterBarHeading="Agencies"
+      handleAdd={handleAdd}
+    >
+      <Col className="d-item">
+        {/* TODO: Swap out with new auto complete after switchover */}
+        <AutoCompleteText
```

Comment:
Doesn't Goeme have the new autocomplete implemented already?

Reply:
Yup, I just wanted to get this up for review as I've been keeping it to myself recently... swapping it out before merge. Just want to get my grouping working first